### PR TITLE
Build config for swedbankpay-designguide

### PR DESCRIPTION
**Pull request by Akswii**
Going to squash merge so all the useless "test" commits are not included.

### CODE REVIEW

File: build/appveyor.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ environment:
   # Github
   access_token:
     secure: IGs25EMrhe0au+uVDbOck6akDzC9ykvJeR0zJjhrT6idNuYOr/JpyGdefQxSlWXk
+  # Codecov
```
Comment (erikhallaraaker) [inline build/appveyor.yml]: You are splitting up the github section of variables.. Why move this?

File: build/appveyor.yml [hunk 2]
```diff
@@ -70,19 +72,19 @@ test_script:
...
+  auth_token:
+    secure: IGs25EMrhe0au+uVDbOck6akDzC9ykvJeR0zJjhrT6idNuYOr/JpyGdefQxSlWXk
+  release: DesignGuide v$(GitVersion_SemVer)
+  tag: $(APPVEYOR_REPO_TAG_NAME)
+  description: $(changelog)
+  artifact: /SwedbankPay.DesignGuide.*\.zip/
+  draft: false
+  prerelease: false
+  on:
+    appveyor_repo_tag: true
```
Comment (erikhallaraaker) [inline build/appveyor.yml]: What is going on here? Did you mean to comment out the entire deploy-step?

File: build/appveyor.yml (hunk 1 above)
Comment (Akswii) [reply build/appveyor.yml]: Just a mistake from when i was moving stuff around, will fix 👌

File: build/appveyor.yml (hunk 2 above)
Comment (Akswii) [reply build/appveyor.yml]: Comments left from previous tests.

File: build/appveyor.yml [hunk 3]
```diff
@@ -70,19 +72,19 @@ test_script:
...
```
Comment (erikhallaraaker) [inline build/appveyor.yml]: You need to fix the indentation as well

**Comment by Akswii** [comment]
> 
> 
> A few comments below. You might also want to rename the files and global javascript object (`px`)

Talked to @hmosvoll, and decided to wait with renaming the global js object for now as there is much uncertainty around how the two different designguides are going to be handled.

**Pull request merged**